cdap-watchdog/src/test/java/io/cdap/cdap/logging/appender/ErrorClassificationLoggingTest.java

```diff
@@ -54,11 +57,11 @@ public class ErrorClassificationLoggingTest {
   private static Injector injector;
   private static TransactionManager txManager;
 
-  @ClassRule
-  public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
+  @Rule
+  public TemporaryFolder TMP_FOLDER = new TemporaryFolder();
 
-  @BeforeClass
-  public static void setUpContext() throws Exception {
+  @Before
+  public void setUpContext() throws Exception {
     injector = LoggingTester.createInjector(TMP_FOLDER);
```

Comment:
Same here.  Why do you want to have a new injector each time?

Note that if you really need it, you can still use 1 TMP_FOLDER (as each injector would get it's own subfolder) and you need to change injector/txManager to non-static

Reply:
removed.